keen/client.py

```diff
@@ -492,6 +492,42 @@ def funnel(self, steps, timeframe=None, timezone=None, max_age=None, all_keys=Fa
 
         return self.api.query("funnel", params, all_keys=all_keys)
 
+    def get_collection(self, collection_name):
+        """
+
+        Returns a description of a collection
+
+        :param event_name: the name of the event to get the collection info from
+        """
+
+        return self.api.get_collection(collection_name)
+
+    def get_collections(self):
+        """
+
+        Returns a description of all collections
+        """
+
+        return self.api.get_all_collections()
+
+    def delete_collection(self, collection):
+        """ Performs a deletion of a collection
+
+
+        :param collection_name: the name of the collection
+        """
+
+        self.api.delete_collection(collection)
+
+    def delete_collections(self, collections):
```

Comment:
I added this function before I fully understood how everything was working. I will go ahead and remove it. Thanks!